pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.black]
+line-length = 120
+target-version = ['py38']
+include = '\.pyi?$'
+exclude = '''
+/(
+    \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | build
+  | dist
+)/
+'''
+
+[tool.mypy]
+python_version = "3.8"
+warn_return_any = true
+warn_unused_configs = true
```

Comment:
From the docs, `warn_unused_configs` is for per module config (when not matching anything)
-> stuff like [[tool.mypy.overrides]] in pyproject.toml.
Since we don't use any here, I think we can drop this one.